diff --git a/OpenBCI_32bit_Library_Definitions.h b/OpenBCI_32bit_Library_Definitions.h
index bc28e1e..0980f88 100644
--- a/OpenBCI_32bit_Library_Definitions.h
+++ b/OpenBCI_32bit_Library_Definitions.h
@@ -16,7 +16,7 @@
 #define OPENBCI_BAUD_RATE_MIN_NO_AVG 200000
 
 // File transmissions
-#define OPENBCI_BOP 'A' // Begining of stream packet
+#define OPENBCI_BOP 0xA0 // Begining of stream packet
 
 //PIN CONNECTIONS
 #define ADS_DRDY 9          // ADS data ready pin
